README.md

```diff
@@ -34,54 +43,48 @@ An example of the provenance generated from this repo is below:
     {
       "name": "binary-linux-amd64",
       "digest": {
-        "sha256": "0ae7e4fa71686538440012ee36a2634dbaa19df2dd16a466f52411fb348bbc4e"
+        "sha256": "7bf2e6ebb97e1bdb669d9df73048247f141e2f8e72ab59f23d456f1bc5a041dc"
       }
     }
   ],
   "predicate": {
     "builder": {
-      "id": "https://github.com/slsa-framework/slsa-github-generator-go/.github/workflows/builder.yml@main"
+      "id": "https://github.com/ianlewis/slsa-github-generator-go/.github/workflows/builder.yml@refs/heads/library"
```

Comment:
let's update this to use slsa-framework instead of ianlewis

Reply:
Yep. Blindly copied and pasted from my branch ;)